specification.md

```diff
@@ -145,10 +163,47 @@ configuration.
 See sections under the [Operation](#operation) section for the details of the
 message sequences.
 
+The WebSocket transport is typically used when it is necessary to have instant
+communication ability from the Server to the Agent without waiting for the Agent
+to poll the Server like it is done when using the HTTP transport (see below).
+
+## Plain HTTP Transport
+
+The second supported transport for OpAMP protocol is plain HTTP connection. The
+Agent is an HTTP client and the Server is an HTTP client server. The Agent makes
+POST requests to the Server. The body of the POST request and response is a
+[binary serialized Protobuf](https://developers.google.com/protocol-buffers/docs/encoding)
+message. The Agent sends AgentToServer Protobuf messages in the request body and
+the Server sends ServerToAgent Protobuf messages in the response body.
+
+OpAMP over HTTP is a synchronous, half-duplex message exchange protocol. The
+Agent initiates an HTTP request when it has an AgentToServer message to deliver.
+The Server responds to each HTTP request with a ServerToAgent message it wants
+to deliver to the Agent. If the Agent has nothing to deliver to the Server the
+Agent MUST periodically poll the Server by sending an AgentToServer message
+where only [instance_uid](#instance_uid) field is set. This gives the Server an
+opportunity to send back in the response any messages that the Server wants to
+deliver to the Agent (such as for example a new remote configuration).
+
+The default polling interval when the Agent does have anything to deliver is 30
+seconds. This polling interval SHOULD be configurable on the Agent.
+
+When using HTTP transport the sequence messages is exactly the same as it is
+when using the WebSocket transport. The only difference is in the timing in the
+situation when the Server wants to send a message to the Agent but the Server
+needs to wait for the Agent to poll the Server and establish an HTTP request
+over which the Server's message can be sent back.
+
+The Agent MUST set "Content-Type: application/x-protobuf" request header when
+using plain HTTP transport. When the Server receives an HTTP request with this
+header set it SHOULD assume this is a plain HTTP transport request, otherwise it
+SHOULD assume this is a WebSocket transport initiation.
+
```

Comment:
Should we add something about compression or perhaps we just assume it's a transport propety?

Reply:
Good question. I think for the response compression we can leave it to be negotiated via Accept-Encoding/Content-Encoding headers.

There is no standard HTTP way to do the negotiation for the request. So, if we want the compression possibility either we mandate its support in the spec or leave it undefined (and maybe say that it is user configurable). For OTLP we chose to mandate "none" and "gzip" in the spec: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md#protocol-details
So maybe we should do the same here.

Reply:
Added to the text.